exercises/parallel-letter-frequency/src/lib.rs

```diff
@@ -1 +1,12 @@
+use std::collections::HashMap;
 
+pub fn frequency(input: &[&str], worker_count: i32) -> HashMap<char, u32> {
```

Comment:
WRT the `worker_count` type: in this case I would probably use `usize`, though I wouldn't chunk the inputs naively. Chunking every `worker_count` letters and sending one char at a time to the workers would be slower than single-threaded, because of the time spent synchronizing a send.

The rationale for using `usize` instead of `u32` is simply that at some level, there needs to be a vector of workers, and the size limit for a vector is the pointer size cap.